sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/SecretKey.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+
+namespace Azure.Security.KeyVault.Keys.Cryptography
+{
+    internal class SecretKey : Key
+    {
+        public SecretKey()
+        {
+            KeyMaterial = new JsonWebKey();
+            KeyMaterial.KeyType = KeyType.Oct;
+            KeyMaterial.KeyOps = new[] { KeyOperation.Encrypt, KeyOperation.Decrypt, KeyOperation.WrapKey, KeyOperation.UnwrapKey };
+        }
+
+        private const string IdPropertyName = "id";
```

Comment:
Nit: I was declaring these at the top. While we don't enforce it, it's typical to put fields at the top.